apps/spam-detection-service/src/index.ts

```diff
@@ -0,0 +1,90 @@
+import express from 'express';
+import cors from 'cors';
+
+const app = express();
+const port = process.env.PORT || 3003;
+
+app.use(cors());
+app.use(express.json());
+
+// Health check endpoint
+app.get('/health', (req, res) => {
+  res.json({ status: 'healthy', service: 'spam-detection' });
+});
+
+// Spam detection endpoint
+app.post('/detect', (req, res) => {
```

Comment:
Can we make the request for this similar to the sentiment-anlysis API it will take an array of items and process them in batches I believe, 

Please reference the sentiment analysis app